test/functional/api/cas/cache_config.py

```diff
@@ -32,6 +32,14 @@ class SeqCutOffPolicy(Enum):
     never = 2
     DEFAULT = full
 
+    @classmethod
+    def from_name(cls, name):
+        for policy, policy_name in SeqCutOffPolicy.__members__.items():
+            if name == policy:
+                return policy_name
```

Comment:
I don't think so, current implementation compares:
name (e.g. "full" obtained from casadm) with policy from enum (e.g. "full", "always"...). 
Comparing name with policy_name would be comparing string to smth like "SeqCutOffPolicy.full".
This method reurns policy_name (enum type, not string) so that it can be assigned to SeqCutOffParameters 'policy' field, which is enum.